### PR TITLE
Filter component

### PR DESCRIPTION
**Pull request by Mohitk391**
Added Filter Component.

Files to check:
1) Filter.js
2) FilterContext.js
3) FilterReducer.js

preview: https://62d51ba7a9a73805fe335bd5--chimerical-tarsier-503bf3.netlify.app/

### CODE REVIEW

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
image

If you click on bed label sofa is getting selected rest is wokring fine

**Comment by Mohitk391** [comment]
Its working fine actually.

File: src/pages/cart/Cart.js [hunk 1]
```diff
@@ -2,84 +2,91 @@ import { NavBar } from "../../components/NavBar/NavBar";
 import "./cart.css";
 import sofa from "../../Assets/images/sofa.jpeg";
```
Comment (MayankKumar10) [inline src/pages/cart/Cart.js:3]: export all your images in an index file and import images from it. So you don't have to write a relative path every time.

File: src/pages/cart/Cart.js [hunk 2]
```diff
@@ -2,84 +2,91 @@ import { NavBar } from "../../components/NavBar/NavBar";
...
+                                <div className="card-header">
+                                    <p className="card-title">Sofa</p>
+                                    <p className="card-sub-title">Furn Easy</p>
                                 </div>
-                                <div class="card-price-text">Rs. 5000</div>
-                                <div class="card-footer flex">
-                                    <div class="card-footer-buttons flex flex-gap-1">
-                                        <button class="btn btn-hover wishlist-button">Add to Wishlist</button>
-                                        <p class="quantity">Quantity: <button class="qty-btn"><i class="fa fa-plus-circle" aria-hidden="true"></i></button> 1 <button class="qty-btn"><i class="fa fa-minus-circle" aria-hidden="true"></i></button></p>
+                                <div className="card-price-text">Rs. 5000</div>
```
Comment (MayankKumar10) [inline src/pages/cart/Cart.js]: take price from JSON data

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestions everything else looks good to me you can merge it.

**Pull request merged**